### DISCUSSION TITLE
How to provide multiple host:port in the JDBC connection string?

### DISCUSSION THREAD

**Discussion by stefanofornari** [ing-bank/cassandra-jdbc-wrapper#40] [Q&A]
Hello, it is mentioned several time the connection string is something like:

jdbc:cassandra://host1--host2--host3:9042/keyspace

What if nodes have different ports? can the below be used?

jdbc:cassandra://host1:port1--host2:port2--host3:port3/keyspace

**Comment by maximevw** [comment]
Hello @stefanofornari,

No, unfortunately, it’s not currently possible to specify different ports for each host.

However, it’s a good idea of feature to implement. So, I will transform this question into an issue as feature request.

**Comment by stefanofornari** [comment]
Thanks, it is very helpful if you have some infra behind NAT.
